Calificativ.js

```diff
@@ -14,4 +14,30 @@
      2: IS
      1: IS
 */
+function catalog(nota) {
+  if (nota > 0 && nota < 5) {
+    return "IS";
+  }
+  if (nota > 4 && nota < 7) {
+    return "S";
+  }
+  if (nota > 6 && nota < 9) {
+    return "B";
+  }
+  if (nota > 8 && nota < 11) {
+    return "FB";
+  } else return "Ai introdus o valoare gresita!";
```

Comment:
Ce nu îmi place e că dacă dai o valoare greșită, trece prin toate if-urile. Mai degrabă aș pune undeva la începutul funcției un test `if(nota < 1 || nota > 10) return "Valoare greșită"`.